CriteoPublisherSdk/Tests/Utility/NetworkManager/CR_NetworkCaptor.m

```diff
@@ -33,13 +33,13 @@ @interface CR_NetworkCaptor ()
 @implementation CR_NetworkCaptor
 
 - (instancetype)initWithNetworkManager:(CR_NetworkManager *)networkManager {
-  MockWKWebView *webView = [[MockWKWebView alloc] init];
-  CR_ThreadManager *threadManager = [[CR_ThreadManager alloc] init];
-  CR_DeviceInfo *deviceInfo = [[CR_DeviceInfo alloc] initWithThreadManager:threadManager
-                                                               testWebView:webView];
-  NSURLSession *session = [NSURLSession
-      sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration]];
-  self = [super initWithDeviceInfo:deviceInfo session:session threadManager:threadManager];
+  // FIXME This is a decorator and network manager should be a protocol. This would allow to not
```

Comment:
When is the FIXME planned to be worked on? JIRA?

Reply:
Added JIRA ticket